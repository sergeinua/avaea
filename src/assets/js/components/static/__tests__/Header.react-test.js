import React from 'react';
import StaticHeader from '../Header';
import renderer from 'react-test-renderer';

test('Header component test', () => {
  var mockUser = {email: "test@gmail.com", id: 58};
  var mockLocation = {pathname: '/home'};
  var component = renderer.create(
    <StaticHeader user={mockUser} location={mockLocation}/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
