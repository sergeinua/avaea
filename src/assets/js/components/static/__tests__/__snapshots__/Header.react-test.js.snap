exports[`test Header component test 1`] = `
<header
  className="static">
  <div
    className="static-nav-button"
    onClick={[Function]} />
  <a
    className="brand"
    onClick={[Function]}
    style={Object {}} />
  <nav>
    <div
      className="left">
      <ul
        className="menu nav">
        <li
          className="">
          <a
            onClick={[Function]}
            style={Object {}}>
            About
          </a>
        </li>
        <li
          className="">
          <a
            onClick={[Function]}
            style={Object {}}>
            Team
          </a>
        </li>
        <li
          className="">
          <a
            onClick={[Function]}
            style={Object {}}>
            Contact
          </a>
        </li>
      </ul>
    </div>
    <div
      className="right">
      <ul
        className="menu user">
        <li
          className="flight-search">
          <a
            className="flight-search"
            onClick={[Function]}
            style={Object {}}>
            Flight Search
          </a>
        </li>
        <li
          className="active">
          <a
            href="/logout"
            onClick={[Function]}>
            Log Out
          </a>
        </li>
      </ul>
    </div>
  </nav>
</header>
`;
